pkg/agent/agent.go

```diff
@@ -90,7 +94,17 @@ func (a *Agent) Run(ctx context.Context) error {
 		return err
 	}
 
-	manager, err := a.newManager(ctx, cat, metrics, as)
+	// If there is an SVID Store plugin, create a buffered pipe
+	var pipeIn pipe.In
+	var pipeOut pipe.Out
+
+	hasStores := len(cat.GetSVIDStores()) > 0
+	if hasStores {
+		pipeIn, pipeOut = pipe.BufferedPipe(1000)
```

Comment:
should buffer size be an agent config? I could see operators having a wide range of typical count of secret store SVIDs they would anticipate handling. Some may be doing SVID Store pushes all day and want channels to be blocked as infrequently as possible, others (as a lesser concern given SVID size) may have very few cases and so not need the additional memory allocation.